0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -199,12 +208,19 @@ Submit a fulfillment to a ledger. Plugin must be connected, otherwise the promis
 
 **TODO**: Define what the message format is.  Plugin must be connected, otherwise the promise should reject.
 
+#### rejectIncomingTransfer
+<code>ledgerPlugin.rejectIncomingTransfer( **transferId**:String ) ⇒ Promise.&lt;null></code>
+
+Reject an incoming transfer that is held pending the fulfillment of its `executionCondition` before the `expiresAt` time.
+
```

Comment:
> Does this mean the protocol doesn't allow ledgers to reject a transfer that is unconditional?

The protocol doesn't allow or disallow any functionality ledgers (and corresponding plugins) choose to implement. This interface specifies the functionality necessary for clients and connectors to use plugins interchangeably for sending and receiving ILP payments.

Right now the "authorized credits" feature is not part of this interface. It could be added, but we need to consider whether this is part of the minimal functionality we expect from all ledgers and plugins.

Note that ledger plugin implementations MAY emit more events and provide more functionality than specified here. However, we should define the interface such that the vast majority will implement all of the minimum functionality described here.

> What should a connector do with an optimistic payment that it can't forward?

Send it back or keep it.